ui/apps/platform/src/Containers/VulnMgmt/widgets/TopRiskiestEntities.js

```diff
@@ -20,7 +20,9 @@ import { WIDGET_PAGINATION_START_OFFSET } from 'constants/workflowPages.constant
 import { entitySortFieldsMap } from 'constants/sortFields';
 import { resourceLabels } from 'messages/common';
 import { entityPriorityField } from 'Containers/VulnMgmt/VulnMgmt.constants';
+import useFeatureFlags from 'hooks/useFeatureFlags';
 
+// TODO: remove once ROX_FRONTEND_VM_UDPATES is enabled
 const TOP_RISKIEST_IMAGES = gql`
     query topRiskiestImages($query: String, $pagination: Pagination) {
         results: images(query: $query, pagination: $pagination) {
```

Comment:
I tried using `results: imageVulnerabilities(...` like I did for `Recently detected image vulnerabilities` but the query did not work. Any suggestions on how to get the data for image cves only?

Reply:
not sure what `sortOption` you're talking about here -- the diff doesn't show that I modified this from the query from what i can tell? i can add it to the apollo `useQuery` if it's necessary though

Reply:
the sort options are already baked into the logic -- you can check by clicking through the `View all` button and checking the URL 
e.g.
`https://localhost:3000/main/vulnerability-management/images?sort[0][id]=Image%20Risk%20Priority&sort[0][desc]=false`